src/components/App/App.js

```diff
@@ -68,6 +77,23 @@ class App extends Component {
     });
   };
 
+  openModal = (title, content, onClose) => {
+    this.setState({
+      modal: {
+        title,
+        content,
+        onClose
+      }
+    })
+  }
+
+  closeModal = () => {
```

Comment:
Is this function needed? Because the component itself is taking care about the closing.

Reply:
oops, I forgot to remove this method, as I implement it on modal itself